src/highrl/policy/feature_extractors.py

```diff
@@ -1,17 +1,19 @@
 """
 Implementation of features exctractors for both the teacher and the robot
 """
-from typing import Optional
-import gym
-import torch.nn as nn
+import logging
+from gym import spaces
+from torch import nn
 import torch as th
 from stable_baselines3.common.torch_layers import BaseFeaturesExtractor
 
+_LOG = logging.getLogger(__name__)
+
 
 class LSTMFeatureExtractor(BaseFeaturesExtractor):
-    """_summary_"""
+    """Input"""
 
-    def __init__(self, observation_space: gym.spaces.Box, features_dim: int = 6):  # type: ignore
+    def __init__(self, observation_space: spaces.Box, features_dim: int = 6):  # type: ignore
         super().__init__(observation_space, features_dim)
         self.LSTM = nn.LSTM(input_size=features_dim, hidden_size=16, num_layers=1)
 
```

Comment:
Where is class being used?
Did not you replace it with the `TeacherFeatureExtractor`

Reply:
Removed. 

However, there must be another pull request set up on top of this one to integrate the new LSTM & env generator into the teacher model.

Reply:
Noted.
I will start working on that PR ASAP.